gyp_xwalk

```diff
@@ -354,6 +354,18 @@ if __name__ == '__main__':
     args.append('-Dnotifications=1')
     args.append('-Drelease_unwind_tables=0')
 
+  parameter = '-Dverify_xwalk_apk='
```

Comment:
My purpose is to make verify_xwalk_apk to be 1 on release mode and to be 0 on debug mode automatically. I removed the -Dverify_xwalk_apk option and tried to insert following code into xwalk.gyp but still got the error message "Undefined variable vefify_xwalk_apk". Could you give any hints about this?

```
'configurations' : {
    'Debug': {
        'variables': {
             'verify_xwalk_apk': 0,
        },
    },
    'Release': {
        'variables': {
             'verify_xwalk_apk': 1,
        },
    },
}